CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ## [0.11.0] - 2023-01-17
 
+### Fixed
+
+- Show `p5` error messages in the user interface (#346)
+
```

Comment:
This needs to be in the `## Unreleased` section (also you've added a blank line above 0.10.0 heading)

Reply:
Oops 😬 Should be fixed now